www/modules/centreon-open-tickets/views/rules/index.php

```diff
@@ -42,34 +42,34 @@
 
 try {
     switch ($o) {
-        case 'a':
-            require_once 'form.php';
-            break;
-        case 'd':
-            $rule->delete($select);
-            require_once 'list.php';
-            break;
-        case 'c':
-            require_once 'form.php';
-            break;
-        case 'l':
-            require_once 'list.php';
-            break;
-        case 'dp':
-            $rule->duplicate($select, $duplicateNb);
-            require_once 'list.php';
-            break;
-        case 'e':
-            $rule->enable($select);
-            require_once 'list.php';
-            break;
-        case 'ds':
-            $rule->disable($select);
-            require_once 'list.php';
-            break;
-        default:
-            require_once 'list.php';
-            break;
+    case 'a':
```

Comment:
wrong indentation -> PSR2 specify that after a switch, a case need a tab -> this will break the jobs on the sonar step